src/test/fixtures/composes.tsx

```diff
@@ -472,6 +489,31 @@ export const mockStatus = (composeId: string): ComposeStatus => {
         ],
       },
     },
+    'hyk93673-8dcc-4a61-ac30-e9f4940d8346': {
+      image_status: {
+        status: 'success',
+        upload_status: {
+          options: {
+            url: 'https://s3.amazonaws.com/b7193673-8dcc-4a5f-ac30-e9f4940d8346-disk.vmdk',
```

Comment:
This might be mostly irrelevant, but can you update the url to reflect the uuid of the image? Could prevent some confusion later even though I can't really recall if I have ever needed to check an url of a mock compose.

Reply:
Nice catch, I think it could definitely cause problems in the future.